python/example_code/rds/lending_library/library_api/chalicelib/library_data.py

```diff
@@ -0,0 +1,397 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+"""
+Purpose
+
+Shows how to use the Amazon RDS Data Service to interact with an Amazon Aurora
+database.
+
+This file is deployed to AWS Lambda as part of the Chalice deployment.
+"""
+
+import datetime
+import logging
+import os
+import boto3
+from botocore.exceptions import ClientError
+from .mysql_helper import Table, Column, ForeignKey
+from .mysql_helper import create_table, insert, update, query, unpack_query_results, \
```

Comment:
style nit:
```python
from .mysql_helper import (
    create_table, insert, update, query, unpack_query_results,
    unpack_insert_results, delete
)
```
There's a push within the community to remove `\` line continuation from Python code where possible.

Reply:
Fixed.